apps/www/public/registry/colors/index.json

```diff
@@ -45,9 +45,9 @@
       "scale": 300,
       "hex": "#cbd5e1",
       "rgb": "rgb(203,213,225)",
-      "hsl": "hsl(212.7,26.8%,83.9)",
+      "hsl": "hsl(212.7,26.8%,83.9%)",
       "rgbChannel": "203 213 225",
-      "hslChannel": "hsl(212.7,26.8%,83.9)"
+      "hslChannel": "212.7,26.8%,83.9%"
```

Comment:
Also here without it should be without commas

Reply:
Okay, I have corrected it.